site/content/kapp-controller/docs/latest/packaging-gitops.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Package Management with GitOps
+---
+
+As you begin working with the [package management APIs](packaging.md) for kapp-controller, you may 
+be wondering how to use kapp-controller's gitops features to manage kapp-controller packages. This 
+section will cover an example gitops workflow using kapp-controller's package management resources.
+
+### GitOps Scenario
+
+An example gitops scenario with kapp-controller could be that a user wants to install a subset of 
+[Packages](packaging.md#package) from a [PackageRepository](packaging.md#package-repository). The 
+user wants to define which PackageRepository and Packages to install by defining these resources in 
+a git repository. This way, a user can manage resources in a declarative fashion and track the history 
+of changes to a Kubernetes cluster. 
+
+After making changes to the main branch of the git repository, the user expects that kapp-controller 
+will sync these resources to the Kubernetes cluster where kapp-controller is installed. The user also 
+expects kapp-controller to sync these resources based on updates (e.g. PackageRepository or Package version upgrades) 
+to the main branch of the git repository.
+
+### Package Management GitOps Example
+
+To start, a user should already have kapp-controller installed on a Kubernetes cluster and have 
+a git repository available. 
+
+First, a user can start by defining an [App custom resource](app-overview.md) like below. **NOTE:** 
+A user will also need to create a serviceaccount with associated RBAC permissions for the App to use.
+
+```yaml
+apiVersion: kappctrl.k14s.io/v1alpha1
+kind: App
+metadata:
+  name: pkg-gitops-example
+  namespace: pkg-gitops
+spec:
+  serviceAccountName: pkg-gitops-app-sa
+  fetch:
+  - git:
+      url: https://github.com/user/my-pkg-gitops-repo
+      ref: origin/main
+
+  template:
+  - ytt: {}
+
+  deploy:
+  - kapp: {}
+```
+
+The App will be pointed at the git repository branch where kapp-controller resources 
```

Comment:
It might be worth adding in a link to the kapp-controller App CR Overview:

https://carvel.dev/kapp-controller/docs/latest/app-overview/#git-authentication

I would probably try gitops out with a private repository so would be nice to see where I can add in those credentials

Reply:
Thanks, added the link to git auth.